tests/test_bench.py

```diff
@@ -365,25 +393,23 @@ def test_unidirectional_coupling_map():
             None,
             None,
         ),
-        (
-            "groundstate",
-            1,
-            4,
-            "small",
-            "qiskit",
-            None,
-            None,
-            None,
-        ),
+        # (
+        #     "groundstate",
+        #     1,
+        #     4,
+        #     "small",
+        #     "qiskit",
+        #     None,
+        #     None,
+        #     None,
+        # ),
```

Comment:
is that removed on purpose?

Reply:
Thanks for noticing, it was accidental. I re-added it.